junos/resource_static_route_test.go

```diff
@@ -38,6 +38,26 @@ func TestAccJunosStaticRoute_basic(t *testing.T) {
 							"community.#", "1"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_instance",
 							"community.0", "no-advertise"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"routing_instance", "testacc_staticRoute"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"preference", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"metric", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"qualified_next_hop.#", "1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"qualified_next_hop.0.next_hop", "192.0.1.1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"qualified_next_hop.0.preference", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"qualified_next_hop.0.metric", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"qualified_next_hop.0.interface", "st0.0"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"community.#", "1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_interface",
+							"community.0", "no-advertise"),
```

Comment:
Unnecessary tests, several arguments already tested. Just add the test of the `interface` argument.

Reply:
Fixed